docs/examples/QuoteInline.jsx

```diff
@@ -0,0 +1,8 @@
+var example = (
+  <div>
+    <p>Gandalf, Captain of the starship enterprise said <UIToolkit.Quote type="inline" author="Gandalf" role="Captain of the Starship Enterprise" cite="The Internet" purpose="success">Use the force Harry</UIToolkit.Quote>
```

Comment:
[`Enterprise`](https://en.wikipedia.org/wiki/USS_Enterprise_%28NCC-1701-D%29) is a proper noun and should be capitalised (disclaimer: I am not a trekkie). Same probably goes for [`the Force`](https://en.wikipedia.org/wiki/The_Force_%28Star_Wars%29).